src/views/router-views/email/update-submitted.njk

```diff
@@ -34,6 +34,11 @@
 
     {% if true === returnToConfirmationStatement %}
         <form action="" method="post">
+            {{
```

Comment:
Non-blocking (just something to consider in the future): Nunjucks style here a bit more consistent with the other Macros having it like"

```njk
{{ csrfTokenInput({ csrfToken: csrfToken }) }}
```

or 


```njk
{{ csrfTokenInput({
    csrfToken: csrfToken 
}) }}
```